components/Map/Map.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import districtJson from '../../geoJSON/district.json';
+import subcountyJson from '../../geoJSON/subcounty.json';
```

Comment:
Why are these in the project root? If they're required for this component, move them into the component folder.

Also, if this component is only for the Uganda map, please rename it accordingly.

Reply:
The `geoJSON` is now only in `Mapcontainer`. This component will be shared by both Uganda and Kenya.